index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
+  <link href='https://fonts.googleapis.com/css?family=Crete Round' rel='stylesheet'>
+  <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
+  <link rel="stylesheet" href="css/styles.css">
+  <title>Sonia Karungi - Portfolio</title>
+</head>
+<body>
+  <header class="header-container">
+    <section class="logo">
+      <p>Sonia Karungi</p>
+    </section>
+    <section class="nav-menu">
+      <div class="menu-icon"></div>
+      <div class="menu-icon"></div>
+      <div class="menu-icon"></div>
+    </section>
+  </header>
+  <section class="headline-section">
+    <h1 class="hey-there">Hey There. I am Sonia</h1>
+    <h2 class="soft-dev">I am a Software Developer</h2>
+    <h3 class="desc">I can help you build a product, feature or website. Look through some of my work and experience! If you like what you
+      see and have a project you need coded, don’t hestiate to contact me.
+    </h3>
+    <section class="contact-icons">
+      <img class="item-icon" src="img/social-media-icons/github.png" alt="GitHub Logo">
+      <img class="item-icon" src="img/social-media-icons/linkedin.png" alt="linkedin Logo">
+      <img class="item-icon" src="img/social-media-icons/twitter.png"  alt="Twitter Logo">
+    </section>
```

Comment:
Kindly change the `section` tag to be `ul` tag and wrap each image with a `li` tag for semantic purposes.